src/sentry/tasks/integrations/github/pr_comment.py

```diff
@@ -145,12 +148,14 @@ def create_or_update_comment(
 
 @instrumented_task(name="sentry.tasks.integrations.github_pr_comments")
 def comment_workflow(pullrequest_id: int, project_id: int):
+    cache_key = DEBOUNCE_PR_COMMENT_CACHE_KEY(pullrequest_id)
+
     gh_repo_id, pr_key, org_id, issue_list = pr_to_issue_query(pullrequest_id)[0]
 
     try:
         organization = Organization.objects.get_from_cache(id=org_id)
     except Organization.DoesNotExist:
```

Comment:
should these fail silently or should we still raise an error?

Reply:
I'd log something but not eror